Lanchonete/Lanchonete/ Features/Menu/Data/MenuManager.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  MenuManager.swift
+//  Lanchonete
+//
+//  Created by vera.dias on 2/4/21.
+//
+
+import Foundation
+
+public class MenuManager {
+    func getSandwichTypes () -> [SandwichType] {
+        return SandwichType.allCases
+    }
+    
+    func getSandwichPrice(type: SandwichType) -> Double {
+        return Sandwich(type: type).price()
```

Comment:
The same here. The consumer already knows the sandwich type, so it could request the price directly, I think